snmplib/transports/snmpTLSBaseDomain.c

```diff
@@ -245,88 +245,39 @@ netsnmp_tlsbase_verify_server_cert(SSL *ssl, _netsnmpTLSBaseData *tlsdata) {
 
     case NO_FINGERPRINT_AVAILABLE:
         if (tlsdata->their_hostname && tlsdata->their_hostname[0] != '\0') {
-            GENERAL_NAMES      *onames;
-            const GENERAL_NAME *oname = NULL;
-            int                 i, j;
-            int                 count;
-            char                buf[SPRINT_MAX_LEN];
-            int                 is_wildcarded = 0;
-            char               *compare_to;
-
-            /* see if the requested hostname has a wildcard prefix */
-            if (strncmp(tlsdata->their_hostname, "*.", 2) == 0) {
-                is_wildcarded = 1;
-                compare_to = tlsdata->their_hostname + 2;
-            } else {
-                compare_to = tlsdata->their_hostname;
+            size_t their_hostname_len = strlen(tlsdata->their_hostname);
```

Comment:
Please leave a blank line between declarations and statements.

Reply:
Thank you for your valuable feedback, I will touch up and keep in mind for future changes as well!